src/main/java/net/sacredlabyrinth/phaed/simpleclans/commands/clan/LeaderCommands.java

```diff
@@ -178,4 +182,30 @@ public void untrust(Player player, Clan clan, @Conditions("same_clan") @Name("me
         trustedInput.setTrusted(false);
         storage.updateClanPlayer(trustedInput);
     }
+
+    @Subcommand("%discord %create")
+    @CommandPermission("simpleclans.leader.discord.create")
+    @Description("{@@command.description.discord.create}")
+    @Conditions("verified")
+    public void discord(Player player, Clan clan) {
+        DiscordHook discordHook = Objects.requireNonNull(chatManager.getDiscordHook(), "DiscordHook cannot be null");
```

Comment:
The players won't know the cause.
A condition for checking if DiscordSRV is installed and the hook is enabled would be better.

Reply:
> The players won't know the cause. A condition for checking if DiscordSRV is installed and the hook is enabled would be better.

I don't see the point of creating a condition: there is only one method right now that uses `getDiscordHook()`, should we create a condition for it?
Isn't it better to check on null and send the message to player? 👀

Reply:
I was thinking of adding another command (remove), but it's not necessary at the moment.